applications/sandbox/systems/physics_test_system.cpp

```diff
@@ -126,73 +120,58 @@ namespace legion::physics
         //stabilityComparisonScene();
```

Comment:
give physics_test_system a quick manual re-review to check for formatting and commented out code. clean up where possible, not gonna review it too harshly because it's a sandbox class

Reply:
some of the added commented code is related to the recursive template bug, would it be okay with you if I addressed this in my next subfeature?

Reply:
That's fine!